.github/workflows/pull-request.yml

```diff
@@ -11,16 +11,16 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Set up JDK 17
+      - name: Set up JDK 21
         uses: actions/setup-java@v3
         with:
           distribution: 'temurin'
-          java-version: 17.0.7
+          java-version: 21.0.3
       
       - name: Set Up Ballerina
         uses: ballerina-platform/setup-ballerina@v1.1.0
         with:
-          version: 2201.10.3
+          version: 2201.11.0
```

Comment:
Instead of hardcoding the lang version, can we extract it from the `gradle.properties` like this. Otherwise, we may have to update this workflow everytime there is a lang release.

https://github.com/ballerina-platform/ballerina-library/blob/main/.github/workflows/pr-build-connector-template.yml#L26C13-L39C1